docs/source/contribute.rst

```diff
@@ -35,7 +35,7 @@ forward on all code in PyU4V.
 
 2. Use ``.format()`` for string concatenation.
 
-.. code-block::
+.. code-block:: Python
```

Comment:
I have no idea how they got to be the incorrect single quote but:
`variable1=’Hello’, variable2=’PyU4V’)`
Should be...
`variable1='Hello', variable2='PyU4V')`